modules/insights-operator-one-time-gather.adoc

```diff
@@ -20,7 +20,7 @@ You must run a gather operation to create an Insights Operator archive.
 +
 [source,yaml]
 ----
-include::https://raw.githubusercontent.com/openshift/insights-operator/release-4.11/docs/gather-job.yaml[]
+include::https://raw.githubusercontent.com/openshift/insights-operator/release-4.12/docs/gather-job.yaml[]
```

Comment:
Probably needs QE

Reply:
@xJustin @JoaoFula Can you please help with who can review this update?

Reply:
Hi, yes, I'm the QE for insights operator, thank you for the ping. 
This might be a dummy question but if the information is changed within `https://raw.githubusercontent.com/openshift/insights-operator/release-4.12/docs/gather-job.yaml[]`, do we have to recompile or do something to the documentation? Or it will just be automatically applied?
This PR receives my LGTM, but noticed a small mistake when going through the linked file

Reply:
Hey @JoaoFula, no need to recompile. The docs automatically import the latest updates from that github file every time they build. So, updating that file is no problem and can be done anytime.

Reply:
Alright, thanks @xJustin 
LGTM